CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to libbpf-rs
+
+### Anyone is welcome to make libbpf-rs better!
+
+Thank you for wanting to contribute to libbpf-rs. We are delighted to have you.
+
+libbpf-rs is an open source project and we love receiving contributions from the
+community! There are many ways to contribute, such as improving the
+documentation, submitting bug reports and feature requests, reviewing code,
+answering questions on PRs, and submitting code that can be merged into
```

Comment:
Nit: `PR` abbreviation here seems kind of unnecessary (and wasn't introduced)? Could we just use the full term here to lower barrier for understanding and reduce cognitive overhead of thinking through what it means in this context?